### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ogsevko**
- [DEMO LINK](https://ogsevko.github.io/layout_enclosures/)
- [TEST REPORT LINK](https://ogsevko.github.io/layout_enclosures/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+  background-color: black;
+}
+
+.outer_box {
+  width: 350px;
+  margin: 20px;
+  margin-bottom: 40px;
```
Comment (irenhh) [inline src/style.css:9]: don't write margin twice if you already mentioned a short note

**Review by irenhh — CHANGES_REQUESTED** [review]
fix indentations

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,23 @@
...
+    <div class="box yellowish">
+      <div class="box grayish">In programming languages, a closure is a technique for implementing lexically scoped name binding in a
+        language with first-class functions. Operationally, a closure is a record storing a function together with an environment.</div>
+    </div>
+  </div>
+  <div class="outer-box box blueish">
+    <div class="box yellowish">
+      <div class="box grayish">У програмуванні замиканням називають підпрограму, що виконується в
+        середовищі, яке містить одну або більше зв’язаних змінних. Підпрограма має доступ до цих змінних під час виконання.</div>
+    </div>
```
Comment (irenhh) [inline src/index.html:22]:
```suggestion
      <div class="box grayish">
        У програмуванні замиканням називають підпрограму, що виконується в
        середовищі, яке містить одну або більше зв’язаних змінних. Підпрограма має доступ до цих змінних під час виконання.
      </div>
    </div>
```

**Review by irenhh — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (ogsevko) [reply src/index.html:22]: What exactly do I have to fix here?

**Review by irenhh — CHANGES_REQUESTED** [review]
usually, you will receive one comment and fix the same mistake everywhere.
You fixed only in the place I added my comment, what about other similar places?

**Comment by ogsevko** [comment]
Sorry. At first I didn't get exactly what fI had to fix and just commited your change via a button right in this thread. Then I realized what was wrong with my code, so I changed all three occasions and committed, pushed (the develop branch) and published it. But it seems that it didn't work.

**Comment by ogsevko** [comment]
Apologies for this mess. I'm still new to git and sometimes it gets confusing. Hope all is good now. Thank you!

**Review by irenhh — APPROVED** [review]

**Pull request closed without merging**